app/services/form_to_resource_service.rb

```diff
@@ -76,6 +77,19 @@ def process_creators(params, resource)
         resource
       end
 
+      ## TODO: Do the right thing with blank form entries
```

Comment:
How do we manage TODOs? Would it be good to file an issue and add the URL here?

Reply:
This is a very short-term TODO. I'm not done with this feature yet, so I'm just reminding myself that I'm not done.